src/p2p/ModeSystemFuncs.ts

```diff
@@ -248,6 +249,7 @@ export function getExpiredRemovedV2(
   txs: P2P.RotationTypes.Txs & P2P.ApoptosisTypes.Txs,
   info: (...msg: string[]) => void
 ): { expired: number; removed: string[] } {
+  console.log('RUNNING GETEXPIREDREMOVEDV2')
```

Comment:
please check out shardus core commit 8e632d4e96b239828ddb5909bd1bacd3ad6182a7
you can follow a similar pattern on how I put system sliced logs behind a logflag with a guid.  note we will refine one more step soon but what is there is a good start.  (search node_rotation_debug)